package.json

```diff
@@ -16,7 +16,7 @@
     "name": "Microsoft Corporation",
     "email": "batchexplorer@microsoft.com"
   },
-  "version": "2.10.0",
+  "version": "2.11.0",
```

Comment:
Why does this version need to be moved to 2.11.0?

Reply:
The version on `master` will always be the next version to be released.